File: codeforces/15.elephant/elephant.cpp
#include <iostream>
/* Author: José Rodolfo (jric2002) */
using namespace std;
int main() {
  unsigned int x, min_num_steps;
  cin >> x;
  if ((x % 5) == 0) {
    min_num_steps = x / 5;
  }
  else {
    min_num_steps = (x / 5) + 1;
  }
  cout << min_num_steps << endl;
  return 0;
}

File: codeforces/15.elephant/elephant.py
# Author: José Rodolfo (jric2002)
x = int(input())
min_num_steps = 0
if ((x % 5) == 0):
  min_num_steps = int(x / 5)
else:
  min_num_steps = int(x / 5) + 1
print(min_num_steps)